src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -1711,6 +1699,7 @@ test_compatible_null_error_pointer (void)
       error, MONGOC_ERROR_PROTOCOL, MONGOC_ERROR_PROTOCOL_BAD_WIRE_VERSION, "");
 
    /* null error pointer is ok */
+   td = mc_tpld_unsafe_get_mutable (client->topology);
```

Comment:
Updating docs on `mongoc_client_command_simple` is a larger task that updating doc-comments on internal APIs. The topology (AFAIK) is not part of the public API, so this would be a doc-comment on an internal component, but attached to an external API, which is confusion.

I have some orthogonal opinions on our current documentation process that I would like to bring up eventually.